temporal-kotlin/build.gradle

```diff
@@ -42,3 +43,27 @@ task registerNamespace(type: JavaExec) {
 
 test.dependsOn 'registerNamespace'
 
+def jacksonVersions = ['2.9.0', "$jacksonVersion".toString()]
+
+testing {
+    suites {
+        jacksonVersions.forEach { jacksonVersion ->
```

Comment:
These tests are not needed since we already have a setup to test multiple versions of dependencies

Reply:
How are these tests compiled/run? As it stands, it is impossible to call `KotlinObjectMapperFactory.new()` in a project that forcefully downgrades to jackson 2.9.0. This test shows that that is now possible.

Reply:
Hm yes looking at more carefully your test is doing something slightly different, we test building and running with different version of our dependencies, but here your testing building with one version, but running with another. Finding some breaking ABI change between versions most likely.

Reply:
This test, or the inclusion of `org.gradle.api.plugins.jvm.JvmTestSuite` appears to be causing some issues with our [feature test](https://github.com/temporalio/features) I haven't had a chance to deep dive as to why. I am fine merging this without this test for now and I'll open an issue to create more thorough testing strategy for breaking catching breaking ABI changes. We should probably do tests like this for more then just `jackson` versions